lib/client/setClientHeader.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+function setClientHeader(method,model,options){
+    options.headers = options.headers || {};
+    options.headers['brisket-client'] = true;
```

Comment:
if the header is required for all ajax calls to the application, it would be better to use $.ajaxSetup to add the header to ajax requests. that way, you can make calls like $.get("/getsomedata").